Gemfile.lock

```diff
@@ -595,8 +595,8 @@ GEM
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.4.3)
-      loofah (~> 2.3)
+    rails-html-sanitizer (1.4.4)
+      loofah (~> 2.19, >= 2.19.1)
```

Comment:
I believe the fixed version of `rails-html-sanitizer` is specific to the new version of loofah we updated earlier this week. I'm not sure if we should make this version range more permissive.